shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/model/RouteCardDataTest.kt

```diff
@@ -646,15 +637,15 @@ class RouteCardDataTest {
                         RouteCardData.Builder(
                             RouteCardData.LineOrRoute.Line(line, setOf(routeB, routeC, routeD)), mapOf(
                                 stopGov.id to RouteCardData.RouteStopDataBuilder(
-                                    stopGov, emptyList(), mapOf(
+                                    stopGov, listOf(westDir, northDir), mapOf(
                                         0 to RouteCardData.LeafBuilder(
                                             directionId = 0,
-                                            routePatterns = listOf(routeDrp1, routeBrp1, routeCrp1 ),
+                                            routePatterns = listOf(routeBrp1, routeCrp1, routeDrp1 ),
```

Comment:
The B route patterns start & end at government center, but that doesn't mean they should be excluded from the list of route patterns at this stop.